apps/jenkins/jenkins/sbox-intsvc/disk.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: "jenkins"
 spec:
   azureDisk:
-    diskURI: /subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/resourceGroups/aks-infra-cftsbox-intsvc-rg/providers/Microsoft.Compute/disks/jenkins-disk
+    diskURI: /subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/resourceGroups/cft-ptlsbox-network-rg/providers/Microsoft.Compute/disks/jenkins-disk
```

Comment:
@darren221980 can disk be moved to disks-ptlsbox-rg ?

Reply:
@thomast1906 RG now updated.